videoengager/js/videoengager.widget.js

```diff
@@ -492,15 +504,8 @@ class VideoEngager {
       document.getElementsByClassName('cx-submit')[0].innerHTML = submitButton;
     };
 
-    this.terminateInteraction = function () {
```

Comment:
Why this is changed? @ipekbayrak

Reply:
If this is used somewhere by the customers you break it,

Reply:
fixed by having both functions